docs/.vitepress/theme/index.js

```diff
@@ -1,10 +1,11 @@
 // You can directly import Vue files in the theme entry
 // VitePress is pre-configured with @vitejs/plugin-vue.
 import Layout from './Layout.vue'
```

Comment:
Just curious, but why do we use vue for this?

Reply:
Afaik, we have to use Vue components in VitePress if we want to use custom components. For example, @jchavarri implemented syntax switching by making a Vue component: https://github.com/melange-re/melange-re.github.io/blob/master/docs/.vitepress/theme/Switch.vue

Reply:
Yes. There's no way around that, at least that I know of. Unless we move away from Vitepress to something React based.